wwwbase/admin/bulkReplaceStructured.php

```diff
@@ -0,0 +1,84 @@
+<?php
+$startMemory = memory_get_usage();
+
+require_once('../../phplib/Core.php');
+User::mustHave(User::PRIV_ADMIN);
+Util::assertNotMirror();
+
+const SEP_GRCONCAT ='␣'; // ␣ = &#8203;
+const SEP_CONCAT = '|';
+
+$objStructured = Session::get('objStructured');
+
+if ($objStructured == null) {
+  //$objStructured = unserialize(rawurldecode(Request::get('objStructured')));
+  $objStructured = explode(',', Request::get('objStructured'));
+  if ($objStructured == null) {
+    $msg = 'Nu am primit niciun parametru pentru a putea afișa lista.';
+    FlashMessage::add($msg, 'danger');
+    Util::redirect('index.php'); // nothing else to do
+  }
+}
+
+DebugInfo::init();
+
+$defs = Model::factory('Definition')
+       ->where_in('status', [Definition::ST_ACTIVE, Definition::ST_HIDDEN])
```

Comment:
Bănuiesc că aici putem renunța la filtrarea după `status`. Teoretic e imposibil să ajungi aici cu o definiții în `ST_PENDING` sau `ST_DELETED`. Dacă totuși se întâmplă, e mai puțin straniu să le arăți, decât să promiți 5 definiții, apoi să arăți doar 4.

Reply:
Corect!